extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -112,9 +112,9 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 	}
 	try {
 		const parentUrn = context.nodeInfo ? await getParentUrn(context) : undefined;
-		const objectType = context.nodeInfo ? context.nodeInfo.nodeType as ObjectManagement.NodeType : (context.connectionProfile.databaseName === '' ? ObjectManagement.NodeType.Server : ObjectManagement.NodeType.Database);
-		const objectName = context.nodeInfo ? context.nodeInfo.label : objectManagementLoc.PropertiesHeader;
-		const objectUrn = context.nodeInfo ? context.nodeInfo!.metadata!.urn : undefined;
+		let objectType = context.nodeInfo ? context.nodeInfo.nodeType as ObjectManagement.NodeType : (context.connectionProfile.databaseName === '' ? ObjectManagement.NodeType.Server : ObjectManagement.NodeType.Database);
+		let objectName = context.nodeInfo ? context.nodeInfo.label : (context.connectionProfile.databaseName === '' ? context.connectionProfile.serverName : context.connectionProfile.databaseName);
+		let objectUrn = context.nodeInfo ? context.nodeInfo!.metadata!.urn : (context.connectionProfile.databaseName === '' ? `Server[@Name='${context.connectionProfile.serverName}']` : `Server/Database[@Name='${context.connectionProfile.databaseName}']`);
```

Comment:
You shouldn't need the name for any of the URNs - just `Server` should be a valid URN as well

Reply:
I need to use the Server/Database[@Name=] format to retrieve the db properties, else I get an error

Reply:
Yeah, I was just talking about the first part of the ternary : `Server[@Name='${context.connectionProfile.serverName}']`

The other half looks fine 😄